docs/topics/note-booster-unvailable-oso.adoc

```diff
@@ -1,3 +1,3 @@
-//IMPORTANT: This booster is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
-*Limitation:* Run this booster on a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
-This booster is not currently available on {OpenShiftOnline} Starter.
+//IMPORTANT: This example is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this example to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
+*Limitation:* Run this example on a {OpenShiftLocal}. You can also use a manual workflow to deploy this example to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
```

Comment:
When we mention "run" or "deploy", ensure that we use "example application" instead of "application".